### ISSUE TITLE
Not receiving order/fills update of conditional order on WebSocket

### ISSUE THREAD

**Issue by xtatus** [ftexchange/ftx#118]
Only receive order update when the conditional order is trigger. Fills or Cancelled updates do not arrive for conditional orders

**Comment by DamiToma** [comment]
This is by design as far as I know, but it's a huge flaw in my opinion. Most bots work with stop orders and it's impossible to guarantee fast speed of execution when we have to constantly poll or perform REST calls to get to know which orders are in place.

**Comment by peyerroger** [comment]
> This is by design as far as I know, but it's a huge flaw in my opinion. Most bots work with stop orders and it's impossible to guarantee fast speed of execution when we have to constantly poll or perform REST calls to get to know which orders are in place.

I fully agree. The way conditional orders work is very cumbersome. Not only on the sockets channel, but also the way the rest api is designed. Would be great if you guys could refactor that part and somehow harmonize it with the rest of the order api. 

If it were up to me, I'd try to clone the order flow of FIX4.4 and implement it on Rest/WebSocket. It's well known and without the flaws of FIX4.2. It nicely handles the order and trade statuses (OrdStatus/ExecType) within a single message.

https://www.onixs.biz/fix-dictionary/4.4/msgtype_8_8.html
